src/components/builder.ts

```diff
@@ -112,6 +112,9 @@ export class Builder {
      * @param rootFile The root file to be compiled.
      */
     async buildWithExternalCommand(command: string, args: string[], pwd: string, rootFile: string | undefined = undefined) {
+        if (rootFile) {
+            this.extension.manager.ignorePdfFile(rootFile)
+        }
```

Comment:
External build command generally do not honour the `%OUTDIR%` placeholder and therefore `tex2pdf` is likely to return the wrong path. I am not sure we can really do anything about it.